corehq/blobs/tests/test_metadata.py

```diff
@@ -54,11 +54,13 @@ def test_save_properties(self):
         self.db.put(BytesIO(b"content"), meta=meta)
         self.assertEqual(get_meta(meta).properties, {"mood": "Vangelis"})
 
-    def test_save_empty_properties(self):
+    def test_save_empty_properties_returns_none(self):
         meta = new_meta()
         self.assertEqual(meta.properties, {})
+
         self.db.put(BytesIO(b"content"), meta=meta)
-        self.assertEqual(get_meta(meta).properties, {})
+        self.assertIsNone(get_meta(meta).properties)
```

Comment:
Ah I see. I definitely looked closely at `from_db_value` so a bit bummed I didn't put this all together. I'm surprised we were content with NullJSONField returning `None` when used with `values_list` previously. I think generally that `test_save_empty_properties` test confused me since it was hard to determine why we expected the value to return an empty dict in one case but None in another case. Now it is unified which I definitely prefer. Thanks for the change!

Reply:
Yeah, this was my fault. I should not have encoded the (incorrect) `values_list` result in the test. Or better yet, should have implemented `from_db_value` back when I originally wrote that code.